project/src/main/puzzle/score-value-label.gd

```diff
@@ -1,10 +1,33 @@
 extends Label
 ## Displays the player's score.
 
+onready var _score_particles := $ScoreParticles
```

Comment:
comments

Reply:
Changed field/node name to `_topout_particles` precluding the need for a comment.